spring-kafka/src/main/java/org/springframework/kafka/support/TopicPartitionInitialOffset.java

```diff
@@ -83,6 +101,26 @@ public TopicPartitionInitialOffset(String topic, int partition, Long initialOffs
 		this.topicPartition = new TopicPartition(topic, partition);
 		this.initialOffset = initialOffset;
 		this.relativeToCurrent = relativeToCurrent;
+		this.position = null;
+	}
+
+	/**
+	 * Construct an instance with the provided initial offset.
+	 * @param topic the topic.
+	 * @param partition the partition.
+	 * @param initialOffset the initial offset.
+	 * @param relativeToCurrent true for the initial offset to be relative to
```

Comment:
These couple params are not relevant any more.
Travis confirm that.
Will fix on merge...